mut/index/SnootyManifest.py

```diff
@@ -159,7 +159,7 @@ def insert_key_values(facet: Facet, prefix: str = ""):
                     sub_facet, key + HIERARCHY_KEY + facet.value + HIERARCHY_KEY
                 )
 
-        def create_facet(facet_entry: object):
+        def create_facet(facet_entry: Any):
```

Comment:
NIT: this is non-blocking as I know this is a high priority change to get out, but would there be a simple to implement concrete type we could use here instead of Any?

Reply:
I don't think without some code restructuring-- the data looks unstructured, so we'd have to use something like `flutter` to get it into a structured form, and that's outside the scope of this PR I think :D

That would be good to do at some point however